### ISSUE TITLE
Wrap `KRadioButton` groups in `KRadioButtonGroup`

### ISSUE THREAD

**Issue by MisRob** [learningequality/kolibri#12596]
🌱 **Are you new to the codebase? Welcome! Please see [the contributing guidelines](https://github.com/learningequality/kolibri/blob/develop/CONTRIBUTING.md).**
 
## Summary

https://github.com/learningequality/kolibri-design-system/pull/650 introduces a new Kolibri Design System component, `KRadioButtonGroup` that ensures expected keyboard navigation in Firefox https://github.com/learningequality/kolibri/issues/10491.

Now we need to update all places in Kolibri to use it.

## References

- Two first places are already updated in https://github.com/learningequality/kolibri/pull/12325
- This will close https://github.com/learningequality/kolibri/issues/10491

## Guidance

- Find all `<KRadioButton>`s and wrap them in `<KRadioButtonGroup>` as described in https://develop--kolibri-design-system.netlify.app/kradiobutton/.
- If the current Kolibri Design System release doesn't yet have `KRadioButtonGroup` available, you can [use devserver-with-kds command](https://kolibri-dev.readthedocs.io/en/develop/howtos/development_with_kds.html) to run Kolibri with your local KDS 

## Acceptance criteria

- [ ] There is no `KRadioButton` left in Kolibri that's not wrapped in `KRadioButtonGroup`
- [ ] There are no regressions in radio button functionality in any of the updated places

**Comment by Yash-KK** [comment]
@MisRob 
I would like to work on this issue.
Can you assign me?

**Comment by elkielki** [comment]
Hi, can I get assigned to this issue?

**Comment by MisRob** [comment]
Hi @elkielki, thank you - yes. Please follow the guidance closely and ask here if anything comes up.

Also note that the newest Kolibri Design System that has `<KRadioButtonGroup>` available is not yet merged to Kolibri's `develop` and we currently have some issue with yarn linking these two repositories. I'd recommend that in your working branch, you change the two `package.json`'s in the very same way as here https://github.com/learningequality/kolibri/pull/12616 and then run `yarn install`. Let us know if there are any issues.

**Comment by adityashibu** [comment]
Hi is the issue still open?

**Comment by elkielki** [comment]
@MisRob Feel free to reassign this issue to @adityashibu. I followed the 'Getting Started' documentation, but I'm stuck on the command `nodeenv -p --node=18.19.0`. For some reason, it's not installing npm.

**Comment by AllanOXDi** [comment]
Hi @elkielki  I have unsigned you and assigned  @adityashibu . Is there any error you are getting from the terminal that  we can look at?

**Comment by adityashibu** [comment]
Hi @AllanOXDi, Just to make it clear, I need to wrap KRadioButton in KRadioButtonGroup right? So something along the lines of:
```
<KRadioButton
            v-for="language in languageCol"
            :key="language.id"
            ref="languageItem"
            v-model="selectedLanguage"
            :buttonValue="language.id"
            :label="language.lang_name"
            :title="language.english_name"
            class="language-name"
 />
```

would become:
```
<KRadioButtonGroup>
    <KRadioButton
                v-for="language in languageCol"
                :key="language.id"
                ref="languageItem"
                v-model="selectedLanguage"
                :buttonValue="language.id"
                :label="language.lang_name"
                :title="language.english_name"
                class="language-name"
     />
<KRadioButtonGroup />
```

Right?

**Comment by MisRob** [comment]
Hi @adityashibu, yes. I'd ask you to study the guidance and all the links in the issue description where you will find plenty of examples of simple use-cases as well as more complex ones, see what's already done, documentation for components you will work with, and understand how to test that it works as expected in Firefox.

As you navigate Kolibri and preview the places you're refactoring, would you always please make a note about how you navigated as a user to a given place, and include this in your pull request description? It will help our QA team to locate where to test. Doesn't need to be elaborate, just few steps to get to each of those places. Thanks a lot.

**Comment by MisRob** [comment]
@elkielki as @AllanOXDi mentioned, we're glad to help you debug your development server setup if you can give us some logs. Seems you're experiencing troubles with a node version managemen - many people have good experience with [Volta](https://volta.sh/).

**Comment by MisRob** [comment]
Note that https://github.com/learningequality/kolibri/pull/12325 was just merged so the two first places are addressed in the latest `develop`. cc @adityashibu

**Comment by adityashibu** [comment]
Sure I'll have a look at it :)

**Comment by MisRob** [comment]
Hi @adityashibu, are you working on this or planning to? Just to know if we should keep the assignment or not.

**Comment by iamshobhraj** [comment]
Hey @MisRob can you assign me this issue?

**Comment by MisRob** [comment]
Hi @iamshobhraj, yes, thank you

**Comment by Maku38** [comment]
Hello , is this issue still open....if yes can u assign me?

**Comment by AllanOXDi** [comment]
Hi @Maku38 , I will assign this to you. Thanks for contributing

**Comment by bjester** [comment]
@Maku38 this issue is currently assigned.

You’re welcome to find [a "help wanted" issue with no assignee](https://github.com/search?q=org%3Alearningequality+is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+no%3Aassignee&type=issues)

**Comment by AllanOXDi** [comment]
Hey @iamshobhraj. Someone is already assigned to this issue. If you want to work on something else, check out these other issues labeled [help wanted" issue with no assignee](https://github.com/search?q=org%3Alearningequality+is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+no%3Aassignee&type=issues)

**Comment by iamshobhraj** [comment]
@MisRob , I opened a pr for this issue

**Comment by AllanOXDi** [comment]
Hi @iamshobhraj,

Thank you for your contribution! Our team will review the PR and provide feedback soon.

**Comment by yashsaraswat2004** [comment]
Is this issue still open?

**Comment by AllanOXDi** [comment]
Hi @yashsaraswat2004  .It's  already assigned. If you want to work on something else, check out these other issues labeled [help wanted" issue with no assignee](https://github.com/search?q=org%3Alearningequality+is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+no%3Aassignee&type=issues)

**Comment by iamshobhraj** [comment]
@MisRob, this issue should have been resolved by PR #12751. Is there anything further that needs to be addressed?

**Comment by MisRob** [comment]
Hi @iamshobhraj, thanks for the notice! Yes this is now closed by https://github.com/learningequality/kolibri/pull/12751
